tests/e2e/core-tests/specs/shopper/front-end-single-product.test.js

```diff
@@ -20,7 +20,6 @@ const simpleProductName = config.get( 'products.simple.name' );
 const runSingleProductPageTest = () => {
 	describe('Single Product Page', () => {
 		beforeAll(async () => {
-			await merchant.login();
 			simplePostIdValue = await createSimpleProduct();
 			await merchant.logout();
```

Comment:
We should be able to remove this line as well.

Reply:
Done removing this unnecessary merchant logout.

Reply:
Thanks!